README.md

```diff
@@ -58,9 +60,8 @@ module.exports = {
       default: 'webpack',
       prod: 'webpack -p',
     },
-    // learn more about concurrently here: https://npm.im/concurrently
-    validate: 'concurrently "nps lint" "nps test" "nps build"',
-    // concurrently script too verbose for your liking? Check out other/EXAMPLES.md!
+    // learn more about npsUtils here: https://npm.im/nps-utils
+    validate: npsUtils.concurrently.nps('lint', 'test', 'build'),
```

Comment:
Funny that it's actually a little longer than before, but I think it's more straightforward :man_shrugging: